### PR TITLE
Added commands created for POC to EC2

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: Internal POC
related to: demisto/etc#22883 

## Description
 Added the ***aws-ec2-delete-subnets***, ***aws-ec2-describe-internet-gateway***, 
 ***aws-ec2-detach-internet-gateway***, ***aws-ec2-delete-internet-gateway***, and 
 ***aws-ec2-delete-vpc*** commands.

### CODE REVIEW

File: Integrations/AWS-EC2/AWS-EC2.py [hunk 1]
```diff
@@ -2636,6 +2636,117 @@ def modify_image_attribute_command(args):
...
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    kwargs = {}
+    if args.get('InternetGatewayId') is not None:
+        kwargs.update({'InternetGatewayId': args.get('InternetGatewayId')})
+    if args.get('VpcId') is not None:
+        kwargs.update({'VpcId': args.get('VpcId')})
+
+    response = client.detach_internet_gateway(**kwargs)
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
```
Comment (yuvalbenshalom) [inline Integrations/AWS-EC2/AWS-EC2.py]: what if it dose not equal 200? what is the handling?(this dose for the other commands also)

Comment (amshamah419) [reply Integrations/AWS-EC2/AWS-EC2.py]: Actually, I'm not sure. I remember there being a reason for it, but I'll check.

**Comment by amshamah419** [comment]
@yuvalbenshalom Docs are a good question. For the codegen integrations, we just include a link. While this wasn't done with codegen, the docs are. Should we do the same as we did for dynamo?

Also, unit tests are pretty tricky with the boto3 package since I can't really mock data, or patch the client.

File: Integrations/AWS-EC2/AWS-EC2.py (hunk 1 above)
Comment (amshamah419) [reply Integrations/AWS-EC2/AWS-EC2.py]: I remember the reason. If there is anything other than a 200, the boto3 client catches the error which gets returned as an exception (which we handle)

**Comment by Itay4** [comment]
Closing this due to changes done in `master` branch.
Please reopen in a new branch.

**Pull request closed without merging**